CONTRIBUTING.md

```diff
@@ -295,14 +295,19 @@ How to submit pull requests
 When submitting pull requests it is expected that they pass the unit tests for formatting.
 The unit tests enforce alphabetization of elements and a consistent formatting to keep merging as clean as possible.
 
-If you want to run the tests before submitting, first install the dependencies. Using `pip` is recommended.
+#### Testing
+
+To run the tests run ``nosetests`` (nose tests) in the root of the repository.
```

Comment:
A separate section is a good idea but maybe some other intro line or about nose https://nose.readthedocs.io/en/latest/index.html? It would be better for uses to have the command after installing dependencies.